_appliance/azure/launch-an-instance.md

```diff
@@ -0,0 +1,221 @@
+
+   - To see the public IP, click on the VM name link. This will show the public IP of the VM.
+   - To see the private IP click on Networking (below SETTINGS on the left side of the screen).
+
+2. Connect to the VM via SSH, using the private key provided for the admin user.
+
+3. Update the password for both the `admin` and the `thoughtspot` users.
+
+   ```
+   $ sudo passwd admin Changing password for user admin
+   $ sudo passwd thoughtspot Changing password for user thoughtspot
+   ```
+
+4. Update the file `/etc/hosts` with all the node IP addresses for the other VMs
+that will be part of the ThoughtSpot cluster.
+
+### Add Storage Disks
+
+1. Go back to the VM and click on it.
+2. Add 2 SSD disks of 1TB each.
+3. Click **Add data disk** and choose **Create disk from the menu**.
+4. Create one mode data disk (demo-disk2) and save them both.
+5. Click **Save** to add the disks to the VM.
+6. Verify that the disks were added by issuing this command in the shell on the VM:
+
+   ```
+   $ lsblk
+   ```
+
+   Which returns results like:
+
+   ```
+   NAME    MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
+   fd0       2:0    1    4K  0 disk
+   sda       8:0    0  200G  0 disk
+   ├─sda1    8:1    0    1G  0 part /mntboot
+   ├─sda2    8:2    0   20G  0 part /
+   ├─sda3    8:3    0   20G  0 part /update
+   └─sda4    8:4    0  159G  0 part /export
+   sdb       8:16   0    1T  0 disk
+   └─sb1     8:17   0    1T  0 part /mnt/resource
+   sdc       8:32   0    1T  0 disk
+   sdd       8:48   0    1T  0 disk
+   sr0      11:0    1  628K  0 rom
+   ```
+
+7. Unmount the temporary disk by issuing:
+
+   ```
+   $ sudo umount /mnt/resource
+   ```
+
+8. Prepare the disks /dev/sdc and /dev/sdd for ThoughtSpot by issuing the command:
+
+   ```
+   $ sudo /usr/local/scaligent/bin/prepare_disks.sh /dev/sdc /dev/sdd
+   ```
+
+   {% include warning.html content="Do not use the disk /dev/sdb. This is reserved
+   for ThoughtSpot use." %}
+
+9. Check the disks status by issuing:
+
+   ```
+   $ df -h
+   ```
+
+10. Repeat the steps in this section for each node in your cluster.
+
+
+### Make network support settings
+
+All the changes in this section must be re-applied each time after a cluster
+is created or updated. If these changes are not present, a reboot of the VMs
+will not have network access.
+
+So when updating these files, keep a backup to copy after any subsequent cluster
+creation or update.
+
+1. Update hostnames for all the nodes by issuing:
+
+   ```
+   $ sudo hostnamectl set-hostname <HOSTNAME>
+   ```
+
+   If you are using a static name, you can issue:
+
+   ```
+   sudo hostnamectl set-hostname <HOSTNAME> --static
+   ```
+
+2. Update `/etc/hosts` with the IP and hostname:
+
+   ```
+   $ sudo vi /etc/sysconfig/network-scripts/ifcfg-eth0
+
+   DEVICE=eth0 ONBOOT=yes BOOTPROTO=dhcp HWADDR=<Add eth0 MAC> TYPE=Ethernet USERCTL=no PEERDNS=yes IPV6INIT=no
+   ```
+
+3. Do not reboot any of the nodes, until these changes are made to each node:
```

Comment:
It would be good put down a brief note under this section 
for why this changes are needed for Azure.
(We don’t do this in other platforms, it seems that these
are Azure specific) 

Reply:
@10000TB All sounds, good, but I don't know why these settings are recommended. If all else looks good to you, I might go ahead and publish this content out now, and then we can add a note later when we find out what exactly to say about this.